shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/schema/model/TableMetaData.java

```diff
@@ -43,6 +43,9 @@
     
     @Getter(AccessLevel.NONE)
     private final List<String> columnNames = new ArrayList<>();
+
+    @Getter(AccessLevel.NONE)
+    private final List<String> indexNames = new ArrayList<>();
```

Comment:
Please do not change `TableMetaData`, we just load, the usage is not need to change.